### PR TITLE
feat: use chainguard for velero

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: cmd/buildtools/utils.go [hunk 1]
```diff
@@ -255,6 +255,38 @@ func GetLatestGitHubTag(ctx context.Context, owner, repo string) (string, error)
 	return tags[0].GetName(), nil
 }
 
+func GetBestGitHubTag(ctx context.Context, owner, repo string, constrants *semver.Constraints) (string, error) {
+	client := github.NewClient(nil)
+	tags, _, err := client.Repositories.ListTags(ctx, owner, repo, &github.ListOptions{})
+	if err != nil {
+		return "", fmt.Errorf("unable to list tags: %w", err)
```
Comment (sgalsaleh) [inline cmd/buildtools/utils.go:262]:
```suggestion
		return "", fmt.Errorf("list tags: %w", err)
```

File: cmd/buildtools/utils.go [hunk 2]
```diff
@@ -255,6 +255,38 @@ func GetLatestGitHubTag(ctx context.Context, owner, repo string) (string, error)
 	return tags[0].GetName(), nil
 }
 
+func GetBestGitHubTag(ctx context.Context, owner, repo string, constrants *semver.Constraints) (string, error) {
```
Comment (sgalsaleh) [inline cmd/buildtools/utils.go:258]: What does "Best" mean? :)

Comment (sgalsaleh) [reply cmd/buildtools/utils.go:258]: baybe just `GetGitHubTag`?

**Review by sgalsaleh — APPROVED** [review]

File: cmd/buildtools/utils.go (hunk 2 above)
Comment (emosbaugh) [reply cmd/buildtools/utils.go:258]: more descriptive would be something like GetGreatestGitHubTag

**Pull request merged**